src/AppBundle/Repository/EventWikiRepository.php

```diff
@@ -243,7 +245,14 @@ public function getPageIds(
             $rqb->setParameter('categoryTitles', $categoryTitles, Connection::PARAM_STR_ARRAY);
         }
 
-        $result = $this->executeQueryBuilder($rqb)->fetchAll(\PDO::FETCH_COLUMN);
+        // If only pages created or edited are being requested, limit based on the presence of a parent revision. This
+        // matches the check done in EventRepository::getEditStats().
+        if (in_array($type, ['created', 'edited'])) {
+            $typeOperator = 'edited' === $type ? '!=' : '=';
+            $rqb->andWhere("rev_parent_id $typeOperator 0");
+        }
+
+         $result = $this->executeQueryBuilder($rqb)->fetchAll(\PDO::FETCH_COLUMN);
```

Comment:
Weird, there's an extra space indent here that's not being complained about by phpcs.